docs/02-guides/customize-the-sitemap.mdx

```diff
@@ -102,43 +189,87 @@ export default createSitemapFetcher(async () => {
   );
 
   return posts.map((post) => ({
-    url: `/todo/${post.id}`,
+    route: `/todo/${post.id}`,
+    changefreq: "monthly",
+    lastmod: new Date(post.updatedAt),
+    priority: 0.5,
+    images: [post.image],
+    data: post, // we add the full post data which allows for custom filter logic
   }));
 });
 ```
 
-#### Registering a Fetcher
+### Filtering sitemap entries from a fetcher
+
+To filter specific entries from a fetcher, you can register a filter in your
+extension definition.
+
+Filters are applied directly to the `FetcherLeaf`, so this allows different
+filters based on different `FetcherLeaf`'s in the same composition.
 
 ```ts title="./extensions/acme/index.ts"
 import { defineExtension } from "@front-commerce/core";
 
 export default defineExtension({
-  name: "acme-extension",
+  name: "acme",
   meta: import.meta,
-  routes: import.meta.url,
   unstable_lifecycleHooks: {
     onServerServicesInit: async (services, request, config) => {
-      services.DI.register("acme", createAcmeDI(config.acme));
-
-      // highlight-start
-      services.Sitemap.registerFetcher(
-        "todoSitemapFetcher",
-        () => import("./sitemap/todo.ts")
+      // We can add the data types through the generic <{ status: boolean }> typing
+      services.Sitemap.registerFetcherFilter<{ status: boolean }>(
+        "AcmeTodo",
+        // The entries returned are only those of the `AcmeTodo` fetcher leaf.
+        async (entries) => {
+          return entries.filter((entry) => entry.data?.status === "published");
+        }
       );
-      // highlight-end
     },
   },
 });
 ```
 
-:::tip
+### Extending the type declarations
+
+To extend these types, you first need to install the `@front-commerce/types`
+package:
```

Comment:
**nitpick:** the "these" here isn't very explicit.

**suggestion:** add an intro sentence clarifying the **_why_** (i.e: why developers might really want to do this).

… I'm not very inspired (and have time) to suggest one for now. Sorry.